assets/l10n/app_en.arb

```diff
@@ -545,6 +553,14 @@
   "@starredMessagesPageTitle": {
     "description": "Title for the page of starred messages."
   },
+  "channelsPageTitle": "Channels",
+  "@channelsPageTitle": {
+    "description": "Title for the page of channels."
+  },
+  "profilePageTitle": "My profile",
+  "@profilePageTitle": {
+    "description": "Title for the page of the logged in user's profile."
```

Comment:
This name and description is inaccurate — the PR doesn't use this string for the title of the page. The page's title is what appears in the app bar, which for this page is the user's name.

Instead a good description would be:
> Label for main-menu button leading to the user's own profile.

And for the name… perhaps `mainMenuMyProfile`.